manifests/poc-data-cluster/prod/spark/iot-packets-silver.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: iot-packets-silver-query
+  namespace: spark
+data:
+  query.sql: |
+    SELECT 
+      gateway_tmst,
+      oui,
+      net_id,
+      rssi,
+      frequency,
+      snr,
+      datarate,
+      region,
+      b58encodeChecked(gateway) as gateway_key,
+      payload_hash,
```

Comment:
all bytes should be base64 encoded strings, as in the other silver table

Reply:
Fixed